### DISCUSSION TITLE
appium-chromedriver is fixed to version 5.6.2, chromedriver 116 missing

### DISCUSSION THREAD

**Discussion by chasperlii** [appium/appium#18993] [Q&A] [answered]
We have following setup:

> driver: uiautomator2@2.29.4
> appium version: v2.0.1

This means our uiautomator2 uses appium-chromedriver@^5.6.1 which means it should get the newest upgrade [5.6.4](https://github.com/appium/appium-chromedriver/releases/tag/v5.6.4). But if i check the `npm-shrinkwrap.json` it uses the version 5.6.2.

How do i upgrade this? I tried with the following
```
appium driver uninstall uiautomator2
appium driver install uiautomator2
```
or restarting appium, but nothing worked. Should it not update itself?

**Comment by jlipps** [comment, marked as answer]
This could very well be expected. All that the uiautomator2 driver's package.json requires is a minimum version of 5.6.1 for appium-chromedriver. 5.6.2 is compatible with this. Probably a new version of the driver should be released with updated minimum dependencies.

**Reply by chasperlii** [reply]
Mhmm okay, but I still find this behaviour a little bit strange...

But you were right! I manually deleted the npm-shrinkwrap file and the node module appium-chromedriver so that it would download the latest version of appium-chromedriver ([v5.6.5](https://github.com/appium/appium-chromedriver/releases/tag/v5.6.5)). 

It worked for devices that are using chromedriver v116, but **didn't** for devices with chromedriver v115.

> No Chromedriver found that can automate Chrome '115.0.5790'

So i reverted to version 5.6.2. I guess we have to wait for the new version like you mentioned!

**Reply by jlipps** [reply]
got a PR to force the update: https://github.com/appium/appium-uiautomator2-driver/pull/646

**Reply by NisStrom** [reply]
This fixes the problem that it downloads the latest Chromedriver (v116), but devices with Chromedriver v115 can no longer be tested. 

> No Chromedriver found that can automate Chrome '115.0.5790'

Before upgrading to appium 2 I believe this was possible (two different driver versions, or beta versions). Is this no longer possible now?

**Reply by jlipps** [reply]
> Before upgrading to appium 2 I believe this was possible (two different driver versions, or beta versions). Is this no longer possible now?

Do you mean two different versions of chromedriver? This should still be possible by using the chromedriver selection caps with different chromedriver executable paths

**Reply by NisStrom** [reply]
Yes! Okay, we will test that.
